website/www/site/assets/scss/_quotes.scss

```diff
@@ -17,9 +17,44 @@
 
 @import "media";
 
+.quote-img-container {
+  display: flex;
+  flex: 1;
+  justify-content: center;
+  align-items: center;
+  width: 100%;
+}
+
+.quote-img {
+  //width: 100%;
```

Comment:
it's better to remove code instead of comment it.

Reply:
done